bleak/backends/p4android/scanner.py

```diff
@@ -7,6 +7,8 @@
 
 from typing_extensions import Literal
 
+import async_timeout
```

Comment:
This can be grouped with the `typing_extensions` import.

Reply:
Adjusted in dfae35b8cba0908e36d1bdd2e3bae9ea21b04731